CRM/Core/BAO/Cache/Psr16.php

```diff
@@ -217,6 +214,17 @@ public static function getLegacyGroups() {
         $groups[] = 'CiviCRM setting Spec';
       }
     }
+    $atomFeedsSettingExtensionStatus = $extensions->getStatus('be.chiro.civi.atomfeeds');
```

Comment:
I feel like you are mixing & matching $atomFeedsSettingExtensionStatus with $multisiteExtensionStatus

Reply:
thanks @eileenmcnaughton fixed